internal/util/util.go

```diff
@@ -21,7 +22,8 @@ import (
 )
 
 var (
-	ShortRequeue = ctrl.Result{RequeueAfter: time.Second * 5}
+	ShortRequeue                 = ctrl.Result{RequeueAfter: time.Second * 5}
+	ErrorUnreadyPlatformOperator = errors.New("encountered an unready platform operator")
```

Comment:
This may be contentious, so let me know. Mainly adding it so that we can do `errors.Is()` in the unit tests.